etl/steps/data/meadow/idmc/2024-08-02/internal_displacement.meta.yml

```diff
@@ -0,0 +1,163 @@
+      share_of_internally_displaced_pop:
+        title: Share of internally displaced people of total population
+        description_short: |-
+          The share of [internally displaced people](#dod:internally-displaced) of all people living in the country. This includes all people who are currently displaced, due to both conflict and violence and disaster and no matter which year they originally got displaced.
+        unit: "%"
+        short_unit: "%"
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+          - {definitions.idmc_definitions.disaster}
+        processing_level: major
+        description_processing: |-
+          - {definitions.processing_notes.population_share}
+          - This figure is the sum of the number of people displaced due to conflict and violence and the number of people displaced due to natural disasters.
+      conflict_stock_displacement:
+        title: Internally displaced people due to conflict
+        description_short: |-
+          [Internally displaced people](#dod:internally-displaced) due to conflict. This includes all people who are currently displaced due to conflict, no matter which year they originally got displaced.
+        unit: people
+        short_unit: ""
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+      share_of_conflict_displaced_pop:
+        title: Share of internally displaced people due to conflict of total population
+        description_short: |-
+          The share of [internally displaced people](#dod:internally-displaced) due to conflict of all people living in the country. This includes all people who are currently displaced due to conflict, no matter which year they originally got displaced.
+        unit: "%"
+        short_unit: "%"
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+        processing_level: major
+        description_processing: |-
+          - {definitions.processing_notes.population_share}
+      disaster_stock_displacement:
+        title: Internally displaced people due to disaster
+        description_short: |-
+          [Internally displaced people](#dod:internally-displaced) due to natural disasters. This includes all people who are currently displaced due to disasters, no matter which year they originally got displaced.
+        unit: people
+        short_unit: ""
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.disaster}
+      share_of_disaster_displaced_pop:
+        title: Share of internally displaced people due to disaster of total population
```

Comment:
same here but again up to you if you want to change it here or later in the charts